packages/ui/src/components/NavBar.tsx

```diff
@@ -0,0 +1,43 @@
+import Link from 'next/link'
+import Logo from './Logo'
+
+interface Props {
+  tabs: { name: string; href: string }[]
+  playerName: string
+  playerLevel: number
+  playerHref?: string
+}
+
+function NavBar({
```

Comment:
the design system v3.0.0 alpha has been released and includes a navbar component -> we could upgrade it in gbl and try out the new components (let's discuss this)